lib/lolcommits/platform.rb

```diff
@@ -0,0 +1,111 @@
+# -*- encoding : utf-8 -*-
+require 'mini_magick'
+
+module Lolcommits
+  class Platform
+    def self.platform
+      if platform_fakeplatform?   then ENV['LOLCOMMITS_FAKEPLATFORM']
+      elsif platform_fakecapture? then 'Fake'
+      elsif platform_mac?         then 'Mac'
+      elsif platform_linux?       then 'Linux'
+      elsif platform_windows?     then 'Windows'
+      elsif platform_cygwin?      then 'Cygwin'
+      else
+        fail 'Unknown / Unsupported Platform.'
+      end
+    end
+
+    def self.platform_mac?
+      RUBY_PLATFORM.to_s.downcase.include?('darwin')
+    end
+
+    def self.platform_linux?
+      RUBY_PLATFORM.to_s.downcase.include?('linux')
+    end
+
+    def self.platform_windows?
+      !!RUBY_PLATFORM.match(/(win|w)32/)
+    end
+
+    def self.platform_cygwin?
+      RUBY_PLATFORM.to_s.downcase.include?('cygwin')
+    end
+
+    def self.platform_fakecapture?
```

Comment:
I believe that would also make it evaluate to true even if someone set the env to `LOLCOMMITS_FAKECAPTURE=false` then though.  In some of my other projects I use a more complex `to_boolean` env parser that looks for all possible variations of true/false/1/0/etc, but I think this one is only set by our test cases..